tests/test_call_hooks.py

```diff
@@ -151,3 +151,10 @@ def test_custom_python_executable(monkeypatch, tmpdir):
             hooks.get_requires_for_build_wheel()
         runner.assert_called_once()
         assert runner.call_args[0][0][0] == 'some-python'
+
+
+def test_setup_py():
+    hooks = get_hooks('setup-py')
+    with modified_env({'PYTHONPATH': BUILDSYS_PKGS}):
+        res = hooks.get_requires_for_build_wheel({})
+    assert res == ['wheel']  # Shold this be ["setuptools", "wheel"]?
```

Comment:
I think this is correct. This is asking the backend if it needs any *extra* packages to make a wheel. The backend here is setuptools, and it must already be available, otherwise calling the hook would fail, so it doesn't need to specify itself.

Reply:
```suggestion
    assert res == ['wheel']
```